packages/atlas/src/components/_crt/CreateTokenDrawer/steps/TokenIssuanceStep/TokenIssuanceStep.utils.ts

```diff
@@ -0,0 +1,113 @@
+import { z } from 'zod'
+
+export const assuranceOptions = [
+  {
+    label: 'Secure',
+    caption:
+      '6 months cliff & 1 year vesting. You won’t receive any tokens now. You will receive 50% of tokens after 6 months of cliff.',
+    value: 'secure',
+  },
+  {
+    label: 'Safe (Default)',
+    caption: 'No cliff & 6 months vesting. You will receive 50% of tokens now.',
+    value: 'safe',
+  },
+  {
+    label: 'Risky',
+    caption: 'No cliff & No vesting. You receive all tokens now.',
+    value: 'risky',
+  },
+  {
+    label: 'Custom',
+    caption: 'Set your own custom cliff, vesting and first payout.',
+    value: 'custom',
+  },
+]
+
+export const vestingOptions = [
+  {
+    value: '0',
+    name: 'No vesting',
+  },
+  {
+    value: '1',
+    name: '1 month',
+  },
+  {
+    value: '3',
+    name: '3 month',
+  },
+  {
+    value: '6',
+    name: '6 month',
+  },
+]
+
+export const cliffOptions = [
+  {
+    value: '0',
+    name: 'No cliff',
+  },
+  {
+    value: '1',
+    name: '1 month',
+  },
+  {
+    value: '3',
+    name: '3 month',
+  },
+  {
+    value: '6',
+    name: '6 month',
+  },
+]
```

Comment:
The vesting values seems too low the Secure type can't be reproduced with a slightly different cliff or part of the tokens available directly.

Shouldn't these be arbitrary number of months instead of a select ?